R/zzz.R

```diff
@@ -13,15 +13,19 @@ register_mlr3 = function(libname, pkgname) {
   # add the learner to the dictionary
   x$add("classif.ctree", LearnerClassifCTree)
   x$add("regr.ctree", LearnerRegrCTree)
+  x$add("classif.mob", LearnerClassifMob)
+  x$add("regr.mob", LearnerRegrMob)
 }
 
-.onLoad = function(libname, pkgname) { # nolint
+.onLoad = function(libname, pkgname) {
+  # nolint
   register_mlr3()
   setHook(packageEvent("mlr3", "onLoad"), function(...) register_mlr3(),
     action = "append")
 }
 
-.onUnload = function(libpath) { # nolint
+.onUnload = function(libpath) {
+  # nolint
```

Comment:
Watch the linebreaking, probably introduced by styler. The issue is already fixed -> update {styler} and correct the placing of `# nolint`.